numerrin_wrapper/numerrin_pool.py

```diff
@@ -0,0 +1,349 @@
+        pmap = {}
+        indx = 0
+        for point in simphonyMesh.iter_points():
+            numerrin.setnode(self.ph, name, indx, point.coordinates)
+            mmap[point.uid] = indx
+            pmap[indx] = point.uid
+            indx = indx+1
+
+        indx = 0
+        emap = {}
+        for edge in simphonyMesh.iter_edges():
+            pind = []
+            for point in edge.points:
+                pind.append(mmap[point])
+            numerrin.setelementtype(self.ph, name, 1, indx, 1)
+            numerrin.setelement(self.ph, name, 1, indx, 0, tuple(pind))
+            mmap[edge.uid] = indx
+            emap[indx] = edge.uid
+            indx = indx+1
+
+        indx = 0
+        fmap = {}
+        for face in simphonyMesh.iter_faces():
+            pind = []
+            for point in face.points:
+                pind.append(mmap[point])
+            face_renode(pind)
+            if len(pind) == 3:
+                numerrin.setelementtype(self.ph, name, 2, indx, 2)
+            else:
+                numerrin.setelementtype(self.ph, name, 2, indx, 3)
+            numerrin.setelement(self.ph, name, 2, indx, 0, tuple(pind))
+            mmap[face.uid] = indx
+            fmap[indx] = face.uid
+            indx = indx+1
+
+        indx = 0
+        cmap = {}
+        for cell in simphonyMesh.iter_cells():
+            pind = []
+            for point in cell.points:
+                pind.append(mmap[point])
+            cell_renode(pind)
+            if len(pind) == 4:
+                numerrin.setelementtype(self.ph, name, 3, indx, 4)
+            elif len(pind) == 6:
+                numerrin.setelementtype(self.ph, name, 3, indx, 6)
+            else:
+                numerrin.setelementtype(self.ph, name, 3, indx, 7)
+            numerrin.setelement(self.ph, name, 3, indx, 0, tuple(pind))
+            mmap[cell.uid] = indx
+            cmap[indx] = cell.uid
+            indx = indx+1
+
+        # add domains
+        numcode = "omega = Domain(" + name + ")\n"
+        boundaries = []
+        elementcode = ""
+        for face in simphonyMesh.iter_faces():
+            if face.data:
+                if CUBA.LABEL in face.data:
+                    fuid = str(mmap[face.uid])
+                    blabel = face.data[CUBA.LABEL]
+                    if blabel not in boundaries:
+                        boundaries.append(blabel)
+                    elementcode += "AddElement(" + name +\
+                                   "domains" + str(blabel) + "," +\
+                                   fuid + "," + fuid + ")\n" +\
+                                   "Face" + fuid + name +\
+                                   "=" + str(blabel) + "\n"
+
+        for boundary in boundaries:
+            numcode += name + "domains" + str(boundary) +\
+                       "= Domain(" + name + ",2,2)\n"
+        numcode += elementcode
+        code = NumerrinCode(self.ph)
+        code.parse_string(numcode)
+        code.execute(1)
```

Comment:
Is the `execute`called in order to check that the imported mesh and the associated `numcode` are valid?
If yes, perhaps a one line comment is in order ...